cmd/kubeadm/app/apis/kubeadm/v1beta1/defaults.go

```diff
@@ -131,24 +131,25 @@ func SetDefaults_Etcd(obj *ClusterConfiguration) {
 	}
 }
 
-// SetDefaults_JoinConfiguration assigns default values to a regular node
-func SetDefaults_JoinConfiguration(obj *JoinConfiguration) {
+// SetDefaultsJoinConfiguration assigns default values to a regular node
+func SetDefaultsJoinConfiguration(obj *JoinConfiguration) {
 	if obj.CACertPath == "" {
 		obj.CACertPath = DefaultCACertPath
 	}
 
-	SetDefaults_JoinControlPlane(obj.ControlPlane)
-	SetDefaults_Discovery(&obj.Discovery)
+	SetDefaultsJoinControlPlane(obj.ControlPlane)
+	SetDefaultsDiscovery(&obj.Discovery)
 }
 
-func SetDefaults_JoinControlPlane(obj *JoinControlPlane) {
+// SetDefaultsJoinControlPlane when controlplane is not null, the native API port is used
```

Comment:
Maybe you can use something like `SetDefaultsJoinControlPlane assigns default values for the API server` wdyt?

Reply:
ok, thanks.